src/server/pfs/server/api_server.go

```diff
@@ -170,6 +175,21 @@ func (a *apiServer) DeleteRepo(ctx context.Context, request *pfs.DeleteRepoReque
 	return &types.Empty{}, nil
 }
 
+// DeleteRepos implements the pfs.DeleteRepo RPC.  It deletes more than one repo at once.
+func (a *apiServer) DeleteRepos(ctx context.Context, request *pfs.DeleteReposRequest) (resp *pfs.DeleteReposResponse, err error) {
+	if len(request.Projects) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "DeleteRepos must specify project(s) whose repos should be deleted.")
+	}
+	repos, err := a.driver.deleteProjectsRepos(ctx, request.Projects)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
super nit - remove newline

Reply:
I kinda liked it: do some stuff, take a breath, return. But no big deal removing it☺